google-cloud-tools-plugin/src/com/google/gct/idea/CloudToolsPluginInfoService.java

```diff
@@ -21,4 +21,5 @@
  * information service across google-cloud-tools-plugin and google-account-plugin
  */
 public interface CloudToolsPluginInfoService extends PluginInfoService {
+  String getClientVersion();
```

Comment:
I don't expect that's going to happen, both the API parameter and format of
the string are CDB defined.  In the future the services will use the user
agent provided by setApplicationName, it just seems like the One Platform
infrastructure is not there yet.

On Thu, Mar 31, 2016 at 10:10 AM, meltsufin <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/CloudToolsPluginInfoService.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/561#discussion_r58059430>
> :
>
> > @@ -21,4 +21,5 @@
> >   * information service across google-cloud-tools-plugin and google-account-plugin
> >   */
> >  public interface CloudToolsPluginInfoService extends PluginInfoService {
> > +  String getClientVersion();
>
> I know it's currently only used for Cloud Debugger, but there's nothing
> really Cloud Debugger specific in the value of the version. Is it possible
> that other APIs in the future will follow suit, and will also require us to
> pass the client version?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/561/files/9570da98876fc95dee48429991005f18fb2fc336#r58059430>
>